typhon/cli.py

```diff
@@ -0,0 +1,96 @@
+"""This module defines the ``typhon`` command line utility"""
+import argparse
+import logging
+import sys
+
+import coloredlogs
+from qtpy.QtWidgets import QApplication
+
+import typhon
+
+logger = logging.getLogger(__name__)
+
+# Argument Parser Setup
+parser = argparse.ArgumentParser(description='Create a TyphonSuite for '
+                                             'device/s stored in a Happi '
+                                             'Database')
+
+parser.add_argument('devices', nargs='*',
+                    help='Device names to load in the TyphonSuite')
+parser.add_argument('--happi-cfg',
+                    help='Location of happi configuration file '
+                         'if not specified by $HAPPI_CFG environment variable')
+parser.add_argument('--version', action='store_true',
+                    help='Current version and location '
+                         'of Typhon installation.')
+parser.add_argument('--debug', action='store_true',
```

Comment:
Having `-v` for verbosity would be slightly more standard

Reply:
✅ Though not going to bother with multiple verbosity flags i.e `-vvv`